injector/LoggerInstance/AdliLogger.py

```
@@ -0,0 +1,79 @@
+        pass
+
+    def logVariable(self, varid, value):
+        '''
+            Logs the given varid and value.         
+        '''
+        self.count += 1
+        self.variableLogCount += 1
+        try:
+            value = json.dumps(value, default=lambda o: o.__dict__ )
+            logger.info(f"# {varid} {value}")
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)